end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -86,6 +88,7 @@ describe('result page screenshot tests', function(){
     before(function(){
         var url = `${CBIOPORTAL_URL}/index.do?tab_index=tab_visualize&cancer_study_list=coadread_tcga_pub&cancer_study_id=coadread_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&Z_SCORE_THRESHOLD=2.0&case_set_id=coadread_tcga_pub_nonhypermut&case_ids=&gene_list=KRAS+NRAS+BRAF&gene_set_choice=user-defined-list&Action=Submit&show_samples=false&`;
         goToUrlAndSetLocalStorage(url);
+        sessionServiceIsEnabled();
```

Comment:
wrong function name?

Reply:
@adamabeshouse good catch - i kinda dislike using that as a function name here because the same name is being used as a property on the frontendConfig object

Reply:
fair enough